src/main/java/stringcalculator/Numbers.java

```diff
@@ -0,0 +1,18 @@
+package stringcalculator;
+
+public final class Numbers {
+
+  private final String[] numbers;
```

Comment:
```suggestion
  private final List<Number> numbers;
```
이렇게 표현되어야 일급컬렉션이 아닐까요?

Reply:
사실 일급 컬렉션을 만들어주려고 했던건 아니었지만, 배열보단 리스트를 쓰는편이 이득이 있을 것 같아서 변경하겠습니다.